pages/cart.html

```diff
@@ -13,35 +13,41 @@
     <nav class="navbar li-shadow">
       <div class="nav-brand">
         <a href="" class="logo-a">
-          <i class="fas fa-suitcase"></i>
+          <img
+            class="logo-s"
+            src="../images/02 - Colorful - Icon Only.png"
+            alt=""
+          />
         </a>
       </div>
       <div class="nav-hamburger">
         <i class="fas fa-bars"></i>
       </div>
       <div class="nav-menu">
         <div class="nav-start">
-          <a href="" class="nav-item link">Home</a>
-          <a href="" class="nav-item link">Product</a>
-          <a href="" class="nav-item link">About Us</a>
-          <a href="" class="nav-item link">Contact Us</a>
+          <a href="../index.html" class="nav-item link">Home</a>
+          <a href="productListing.html" class="nav-item link">Shop Now</a>
         </div>
         <div class="nav-end">
           <div class="form-div nav-search m-x-3">
             <i class="fas fa-search is-blue"></i>
             <input type="text" class="form-input br-3" placeholder="search" />
           </div>
-          <button class="btn-icon nav-icons m-x-1">
-            <i class="fas fa-shopping-cart"></i>
-          </button>
-          <button class="btn-icon nav-icons m-x-1">
-            <i class="fas fa-bell"></i>
-          </button>
-          <button class="btn-icon nav-icons m-x-1">
-            <i class="fas fa-heart"></i>
-          </button>
-          <button class="btn-secondary nav-btn btn-small">Sign up</button>
-          <button class="btn-primary nav-btn btn-small">Log in</button>
+          <a href="../pages/cart.html" class=""
+            ><button class="btn-icon nav-icons m-x-1">
+              <i class="fas is-dark fa-shopping-cart"></i></button
+          ></a>
+          <a href="../pages/wishlist.html" class="">
```

Comment:
cleanup empty class attribute if not used

Reply:
Yup